packages/audit/scripts/dashboard_stats.py

```
@@ -52,12 +52,15 @@ def calc_cli_stats(events: list[dict]) -> dict:
         events: 集計対象のイベントリスト。
 
     Returns:
-        total / codex / gemini / success_rate / errors_by_type を含む辞書。
+        total / codex / gemini / antigravity / by_tool / success_rate /
+        errors_by_type を含む辞書。`by_tool` は全ツールの内訳（Counter 化）で、
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)